Nanoleaf-wpf/Api/Effects/AmbilightEffect.cs

```diff
@@ -42,29 +41,26 @@ private async Task SetColor()
         {
             var color = ScreenGrabber.GetColor();
 
-            if (color != null)
```

Comment:
This if needs to stay

Reply:
The System.Drawing.Color is a struct, this can't be null.